src/main/java/com/google/flourbot/api/StubClient.java

```diff
@@ -0,0 +1,14 @@
+package com.google.flourbot.api;
+
+public class StubClient implements CloudDocClient{
```

Comment:
Interesting, what is the advantage of stubs over mocks?

Reply:
It was taught in one of the boarding training workshops for STEP. I guess the only advantage is learning how Mockito is maybe implemented behind the scene.